src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx

```diff
@@ -59,31 +59,40 @@ export const Detail = () => {
   const dispatch = useDispatch()
   const { id } = useParams()
 
-  const { get, data, isLoading, patch, del, isSuccess, error } =
+  const { get, data, isLoading, patch, post, del, isSuccess, error } =
     useFetch<StandardTextDetailData>()
 
-  const title = 'Standaardtekst wijzigen'
+  const title = id ? 'Standaardtekst wijzigen' : 'Standaardtekst toevoegen'
   const redirectURL = '..'
 
   const defaultValues: StandardTextForm | null = useMemo(() => {
-    if (!data) return null
-
-    return {
-      categories: data.categories,
-      state: data.state,
-      title: data.title,
-      text: data.text,
-      active: Boolean(data.active),
+    if (id && data) {
+      return {
+        categories: data.categories,
+        state: data.state,
+        title: data.title,
+        text: data.text,
+        active: Boolean(data.active),
```

Comment:
why not destruct data here? and change active?!

Reply:
What benefits does destructuring yield? It doesn't make it shorter.